about/instructions.md

```diff
@@ -18,12 +18,26 @@ meta data
 cross reference: chapter
 :     We created an add-on in the last chapter {doc}`volto_custom_addon`.
 
-cross reference: anchor  
-: create anchor
+cross reference: section
+: **Using headers as anchors**
+
+  If you have a page at myfolder/mypage.md (relative to your documentation root) with the following structure:
+
+      # Title
+
+      ## My Subtitle
+
+  Then the autosectionlabel feature will allow you to reference the section headers like so:
+
+      {ref}`path/to/file_1:My Subtitle`
```

Comment:
Will we have to rewrite all our references to use this syntax, or will they be OK as is?